### PR TITLE
🌱 Add Jont828 to clusterctl reviewers

### PR DESCRIPTION
**Pull request by Jont828**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

**Comment by CecileRobertMichon** [comment]
🎉 

+1

File: OWNERS_ALIASES
Comment (CecileRobertMichon) [inline OWNERS_ALIASES:75]: nit: list should be alphabetically ordered

**Review by ykakarap — COMMENTED** [review]
+1
Great addition! Thanks for all the work you have been doing 😃

**Comment by sbueringer** [comment]
/lgtm

Thank you very much for the continous work you're doing on core Cluster API as well as on related projects like the visualizer and the addon provider!!

**Review by killianmuldoon — APPROVED** [review]
/lgtm

Very welcome addition :partying_face:   Thanks for all your contributions!

**Comment by fabriziopandini** [comment]
Yeeess!! Thanks, @Jont828  for the work done so far and for stepping up for being a reviewer!
/lgtm

+1 to celebrate this at this week office hours and then merge/set a lazy consensus EOW

**Comment by sbueringer** [comment]
Wasn't brought up in the office hours yesterday

**Comment by Jont828** [comment]
My bad, I'll bring it up next week at office hours then!

**Comment by sbueringer** [comment]
Added to the agenda for today.

I can bring it up but feel free to take over the agenda item if you want.

**Comment by sbueringer** [comment]
As discussed in the office hours on 16th November lazy consensus until 16th November.

If you have any objections, please raise them until 16th November, otherwise we merge this PR.

**Comment by sbueringer** [comment]
/hold cancel

Welcome!

**Pull request merged**
